src/Uno.UI/UI/Xaml/Media/Animation/Animators/GPUFloatValueAnimator.iOSmacOS.cs

```diff
@@ -412,7 +412,7 @@ private void FinalizeAnimation()
 				this.Log().DebugFormat("Finalizing animation for GPU Float value animator on property {0}.", _bindingPath.LastOrDefault().PropertyName);
 			}
 
-			if (_valueAnimator.IsRunning)
+			if (_valueAnimator?.IsRunning == true)
```

Comment:
```suggestion
			if (_valueAnimator?.IsRunning ?? false)
```

I found this much easier to read :)

Reply:
There was a discussion about this earlier this week looking to find which one was easier to read and it's true that for the ones who know what "??" means/do this seems to be more readable but overall people reacted saying that "anyValue==true" is easier to be read as it explicitly indicates what is expected (the value to be true).

Reply:
I agree with andres :) == true is clearer for me at least =D

Reply:
I'll add that it helps new folks without a lot of experience in C#, they feel more confident and less intimidated.

Reply:
The problem here is the fact that `_valueAnimator?.IsRunning` is a `int?` and that making this test with `== false` would lead to invalid results if `_valueAnimator` is null.

Even if null coalescing is more complex to understand at first, once the syntax is learnt, it avoids errors leading to understanding how default value for nullable expressions works, when generalizing this pattern.

Moreover, enabling nullable ref types in this type of scenario would avoid additional errors.

Reply:
Isn't IsRunning a boolean?

Reply:
Right ;) Typo, yet still the same point.